### ISSUE TITLE
env vars not available in windows

### ISSUE THREAD

**Issue by rstiller-goeuro** [goeuro/myke#69]
tested myke on windows 10 pro with `cygwin`, `PowerShell` and `CMD` - no env vars availabe.

* myke.env present
* myke.env.local present
* env section ind myke.yml present

but none of the env vars is available.

Env vars in Windows 10 Pro are only availbe within the ubuntu linux bash layer ...

*myke.yml*
```yml
---
project: booking-system
env_files:
  - default-values.env
env:
  ABC: default
  PATH: path_from_yml
tasks:
  echo:
    cmd: "echo 123 $ABC def"
  test:
    cmd: env
    before: $myke echo
```

*myke.env*
```bash
ABC=abc
```

*myke.env.local*
```bash
ABC=def
```

*output*
```bash
$ myke test
booking-system/test: Running
+ echo

+ env
PWD=/cygdrive/c/Users/goeuro/projects/goeuro/booking-system/myke
HOME=/cygdrive/c/Users/goeuro
TERM=cygwin
SHLVL=1
_=/usr/bin/env
SYSTEMROOT=C:\WINDOWS
WINDIR=C:\WINDOWS
booking-system/test: Completed, Took: 80.0767ms
```

**Comment by rdsubhas** [comment]
Interesting, I don't have a windows machine, so lets check this out tomorrow

**Comment by rdsubhas** [comment]
Fixed by test cases in #74, thanks
